osgp/platform/osgp-core/src/main/resources/db/migration/V20190724133425000__Adds_connection_details_to_device_table.sql

```diff
@@ -0,0 +1,22 @@
+DO
+$$
+BEGIN
+
+  IF NOT EXISTS(SELECT 1 FROM information_schema.columns WHERE table_schema = current_schema AND table_name = 'device' AND column_name = 'last_successful_connection_timestamp')
+  THEN
+    ALTER TABLE device ADD COLUMN last_successful_connection_timestamp timestamp without time zone;
+    ALTER TABLE device ADD COLUMN last_failed_connection_timestamp timestamp without time zone;
+    ALTER TABLE device ADD COLUMN failed_connection_count INT;
```

Comment:
Make this: 
`ALTER TABLE device ADD COLUMN failed_connection_count INT DEFAULT 0 NOT NULL;`
And drop the next two (non-empty) lines.